src/Commands/ImportDump.php

```diff
@@ -234,10 +234,20 @@ public function handle()
             }
         }
 
-        if ($importFilePath && ($optionForce || $this->confirm(sprintf('Are you sure that you want to import the dump at %s?', $disk->path($importFilePath))))) {
+        $this->info(sprintf('Dump: "%s"', $disk->path($importFilePath)));
```

Comment:
Why does the dump file that the user has chosen just before, have to be printed out again here?

Reply:
it was already printed out again in the confirmation question. this is not a change, it just rearranged that information on the screen. also it has the advantage of showing the file in non-interactive mode.